package.json

```diff
@@ -80,9 +80,21 @@
     "lint": "node_modules/.bin/esw webpack.config.* app",
     "lint:watch": "npm run lint -- --watch",
     "test": "node_modules/.bin/mocha tests/**/*.test.* --require tests/setup.js",
-    "coverage": "rimraf coverage && istanbul cover _mocha tests/**/*.test.* -- --require tests/setUp.js -R spec",
+    "coverage": "nyc -all -e .jsx npm test",
     "coveralls": "npm run coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js"
   },
+  "nyc": {
```

Comment:
I am a little curious about this implementation. I see here you have excluded tests files. Yet up there on `coverage` it runs the nyc package  and runs the `npm test` command?

It seems quite redundant to exclude the test files so you can include them in the test section. 

I guess my question is, why did you opt out of the previous implementation?

Reply:
The previous implementation was wrongly configured and was getting the percentage of a single file and I tried to get the workaround to cover all the coverage but the most recommendation was to use `nyc` which is also a stable implementation of Istanbul.

What am excluding from the `nyc` config object is the test coverage of the tests files themselves since I don't want to get their coverage.